projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployer.java

```diff
@@ -70,6 +70,10 @@ public Optional<JobDeploymentStatus> readScheduledJob(String dataJobName) {
     return jobDeployment;
   }
 
+  public boolean deploymentExists(String dataJobName) {
+    return dataJobsKubernetesService.cronJobExists(getCronJobName(dataJobName));
```

Comment:
nit: 
(1) unexpected naming; the above method `readScheduledJob` also uses the cronjob, why not scheduledJobExists to keep consistency of terminology? 

(2) why using `getCronJobName`? 
```
// Public for integration testing purposes
  public static String getCronJobName(String jobName) {
    return jobName;
  }
```
(3) Why do we need this delegator to `dataJobsKubernetesService.cronJobExists`? It simply links a feature of another service, why is that needed?